vmware/go-vcloud-director
# Add vendor directory to simplify dependencies

Related Issue: #47.

The vendor directory includes our project direct dependencies (check.v1,
yaml.v2) and a dependency of yaml (pretty).

The vendor directory has been created using govendor
(https://github.com/kardianos/govendor)

The file vendor/vendor.json is the index. While it is not needed for
builds, it will simplify future changes to the vendor folder.

There are no changes to existing files in this PR. Only the ones in the vendor directory.

How to test it:

(test n. 1)
from your `$GOPATH` directory, delete the following:
* `github.com/kr/pretty`
* `gopkg.in/check.v1`
* `gopkg.in/yaml.v2`

Then build and run the tests as usual. The library will be compiled using the files in the vendor directory.

(test n. 2)
Instead of deleting the above packages, edit one or more files in them and introduce a syntax error.
Then build and run the tests. The faulty files will be ignored, and the ones under vendor will be used instead.

Signed-off-by: Giuseppe Maxia
